src/trust/types.ts

```diff
@@ -35,21 +50,23 @@ const CredentialDefinitionMetadata = z.object({
   type: z.array(z.string()),
   credentialSubject: z.record(
     z.object({
-      mandatory: z.boolean(),
+      value_type: z.string(),
       display: z.array(z.object({ name: z.string(), locale: z.string() })),
+      mandatory: z.enum(["true", "false"]),
```

Comment:
Sa is a string and not a boolean there should be an error in the API. I would leave it boolean and not adapt it to the response.

Reply:
Removed in a6639c0aaebed8c026d8e1dc2ab377e9cc27e8b2, I forgot to change it back after testing it. Thanks.